flytekit/models/task.py

```diff
@@ -872,6 +874,14 @@ def image(self):
         """
         return self._image
 
+    @property
+    def architecture(self):
+        """
+        Architecures supported by the container's image.
+        :rtype: Text
```

Comment:
shouldn't this be int?  Also we can just remove this line now that we have mypy style hints.  https://github.com/flyteorg/flyteidl/pull/226/files#diff-8bd2613bf781125f77247c29a2f5502658c63cccf2b597a9ec3695ef1e7b2ca9R188

Reply:
Updated with int